config.go

```diff
@@ -579,3 +579,30 @@ func (b *Builder) SetDomainname(name string) {
 func (b *Builder) SetDefaultMountsFilePath(path string) {
 	b.DefaultMountsFilePath = path
 }
+
+// Comment returns the comment which will be set in the container and in
+//containers built using images built from the container
```

Comment:
Add a whitespace after `//` here, to match the rest.

Reply:
fixed